### ISSUE TITLE
disabledPackageSources 'value' attribute is ignored

### ISSUE THREAD

**Issue by mmitche** [NuGet/Home#10291]
Please read the following information before posting the issue.

## Details about Problem

All versions of nuget, didn't work before.

## Detailed repro steps so we can see the same problem

1. Add a nuget source
2. Place it in disabledPackageSources, but with the "value" attribute set to false (should not be disabled)
3. Attempt restore. Source is still disabled.

**Comment by marsop** [comment]
I am experiencing this problem as well. Basically the `value` parameter is ignored, making it useless.

**Comment by The-Futurist** [comment]
Yes, I just started a discussion about this, it seems this is a long standing bug since people probably rarely set it to "false" and it always behaves as if it was set to "true".

https://github.com/NuGet/Home/discussions/11280

**Comment by DamianEdwards** [comment]
I just hit this myself

**Comment by zivkan** [comment]
As per our [triage policy](https://github.com/NuGet/Home/blob/dev/Issue-Triage-Policy.md#up-voting-a-feature-requestbug), it helps us prioritize if the first/original comment gets a 👍 reaction (upvote), since GitHub lacks other functionality to help track impact (vs a lively discussion).

**Comment by sebastienros** [comment]
Hey @DamianEdwards, nice to see you there! Or not ...
